marge/app.py

```diff
@@ -94,10 +104,16 @@ def regexp(s):
         help="Only process projects that match; e.g. 'some_group/.*' or '(?!exclude/me)'",
     )
     arg(
-        '--max-ci-time-in-minutes',
-        type=int,
-        default=15,
-        help='How long to wait for CI to pass.',
+        '--ci-timeout',
+        type=time_interval,
+        default=time_interval('15min'),
+        help='How long to wait for CI to pass (defaut: 15min).',
+    )
+    arg(
```

Comment:
@aschmolck I understand why you want to be consistent here, but simply dropping the old flag is unnecessarily harsh on users: most of them will run it directly from docker, probably using the `latest` tag, so next time it is restarted, a new version will be pulled, and suddenly their installation will not work anymore, it will take time to figure out why, etc. 

I would suggest keeping the old flag as well, maybe printing something along the lines of "this option is deprecated, use --ci-timeout instead".

Reply:
I thought p(does_not_use_defaul)*user_base was probably small enough, but you're right it's nicer to err on site of caution.